build-tools/build-indexes

```diff
@@ -555,6 +600,9 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 			if (isLetsGo) {
 				return ["Uber", "OU", "UU", "NFE", "LC"];
 			}
+			if (isPMOTS) {
+				return ["OU"];
+			}
```

Comment:
this isnt necessary, it uses the default gen 8 table (which includes OU)

Reply:
In Alternatium's case, it only contains OU mons, so this is necessary.

Reply:
No it’s not, look at how nfe does it